spec/lib/authorities/loc_spec.rb

```diff
@@ -26,17 +26,27 @@
     let :authority do
       described_class.subauthority_for("subjects")
     end
+    let(:url) { 'http://id.loc.gov/search/?q=foo&q=cs%3Ahttp%3A%2F%2Fid.loc.gov%2Fauthorities%2Fsubjects&format=json' }
 
     context "for searching" do
       it "returns a url" do
-        url = 'http://id.loc.gov/search/?q=foo&q=cs%3Ahttp%3A%2F%2Fid.loc.gov%2Fauthorities%2Fsubjects&format=json'
         expect(authority.build_query_url("foo")).to eq(url)
       end
     end
 
+    context "with flat params encoded" do
```

Comment:
Can you move this to it's own describe block that is describing the `#response` method?